src/terminal/parser/telemetry.hpp

```diff
@@ -97,6 +97,7 @@ namespace Microsoft::Console::VirtualTerminal
             OSCFG,
             OSCBG,
             DECALN,
+            OSCSCB,

```

Comment:
I believe you also need to update the `TermTelemetry` class with the new telemetry entry. See here:
https://github.com/microsoft/terminal/blob/96a77cb74b4ce2a13facf25daf5d8fd6234c19f9/src/terminal/parser/telemetry.cpp#L269-L270

Reply:
Indeed! This is the last thing I'll block on, and then this is totally ready.